samples/Basic-TFRecorder-Usage.ipynb

```diff
@@ -13,9 +13,17 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
+   "execution_count": 2,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Error importing tfx_bsl_extension.arrow.array_util. Some tfx_bsl functionalities are not available"
```

Comment:
Broken dependency?

Reply:
I'm not sure. Maybe I need to update my packages. I'm still running on Py3.6.

Reply:
Reinstalling tfx-bsl seems to have made it go away.